tests/test65.c

```diff
@@ -16,19 +16,6 @@ struct CacheEntry {
 };
 struct CacheEntry *cache = NULL;
 
-static char * /*value*/ find_in_cache(const char *key)
```

Comment:
On closer inspection, this seems like useful example code. If you're trying to silence a compiler warning about an unused `static` function, how about just removing the `static` keyword, or adding a call to this function in `main`?